src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -120,24 +120,24 @@ public String getApiUri() {
     @Override public void visitSources(SCMSourceObserver observer) throws IOException, InterruptedException {
         TaskListener listener = observer.getListener();
         if (repoOwner.isEmpty()) {
-            listener.getLogger().format("Must specify user or organization%n");
-            return;
+            throw new AbortException("Must specify user or organization%n");
```

Comment:
@jglick This is a UX matter. I prefer the behavior proposed here.

From my PoV an empty `Owner` would be considered a human error and this PR provides a validation to the input fields, in this case, `Owner`.

If the use wants to remove all jobs, he can directly remove his `OrganizationFolder` or his `MultibranchProject`.